src/browser.ts

```diff
@@ -1,4 +1,7 @@
-/** Defines client/browser-specific variables that do not change over the course of a session. */
+/**
+ * Defines client/browser-specific variables that do not change over the course of a session.
+ *
+ */
```

Comment:
Out of curiosity, is the expanded comment format required for the docs to render correctly, or is it just preference?

Reply:
I haven't yet cleaned all files after removing @packageDocumentation. We can write comments in one line.